.github/workflows/ci.yml

```diff
@@ -351,8 +352,8 @@ jobs:
 
       - name: Install MSRV toolchain
         run: |
-          rustup toolchain install ${{ env.MSRV }} --no-self-update --profile=minimal --component clippy
-          rustup default ${{ env.MSRV }}
+          rustup toolchain install ${{ env.DENO_MSRV }} --no-self-update --profile=minimal --component clippy
```

Comment:
I don't think this is going to be sufficient.

The PR for testing frameworks also needs an msrv bump, I'm going to raise an PR which bumps MSRV for everything except core/hal/types.

Reply:
#4204 is landed, will rebase on top of this.